### DISCUSSION TITLE
Future Features

### DISCUSSION THREAD

**Discussion by haskins-io** [haskins-io/Support#1] [Announcements]
Just a quick note to state what I'm currently planning to include in the next release (v1.6.0), hopefully I'll get these done,
but there is no guarantee.

### Drawing Game
A player is given a word, and 30 seconds, and they have to draw a representation of the word within the time limit.

### Find the Lady
Just a bit of fun this one. There are three cards that get shuffled and the player has to guess where the Queen ends up.

## Quiz Improvements
Looking to add a feature to distort images in some way to provide a 'What/Who is this style question'

**Comment by rjRostana** [comment]
Mark

have tried to use the Contact form on your webpage. Gives an error.

**Reply by haskins-io** [reply]
Thanks for pointing that out.

There does seem to be a problem with the form stating that there has been a problem. All my tests have reached me so I'm not sure what is happening there. 

If you want to reach out to me you can either leave a message here or send a message to mark at my domain
